### PR TITLE
Implement logging to kernel log

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/chain.pp [hunk 1]
```diff
@@ -2,9 +2,11 @@
 # @param policy [Ferm::Policies] Set the default policy for a CHAIN
 # @param disable_conntrack [Boolean] disable/enable usage of conntrack
 # @param chain [Ferm::Chains] name of the chain that should be managed
+# @param log_dropped_packets [Boolean] boolean to enable/disable logging of packets to the kernel log, if no explicit chain matched
```
Comment (alexjfisher) [inline manifests/chain.pp]: Is the data type redundant here?  (ie shouldn't it automatically get picked up from the type used below?)

Comment (bastelfreak) [reply manifests/chain.pp]: fixed it

File: manifests/chain.pp [hunk 2]
```diff
@@ -24,4 +26,11 @@
     ),
     order   => '01',
   }
+
+  if $log_dropped_packets {
+    concat::fragment{"${chain}-footer":
```
Comment (alexjfisher) [inline manifests/chain.pp]: Does this need an `order` parameter?

Comment (alexjfisher) [reply manifests/chain.pp]: i guess default is `10`, so it should work?  Maybe make it super obvious that it's the footer by setting it explicitly to something bigger than 10 like 100???

**Pull request merged**